packages/ocs/modals/osd-migration/osd-migration-status.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { DOC_LINKS, ViewDocumentation } from '@odf/mco/utils';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+} from '@odf/shared/status/icons';
+import { CephClusterKind } from '@odf/shared/types';
+import { useCustomTranslation } from '@odf/shared/useCustomTranslationHook';
+import { Button, Flex, FlexItem } from '@patternfly/react-core';
+import { InProgressIcon } from '@patternfly/react-icons';
+import {
+  COMPLETED,
+  PENDING,
+  FAILED,
+  BLUESTORE,
+  BLUESTORE_RDR,
+  getOSDMigrationStatus,
+} from '../../utils/osd-migration';
+
+const calculateOSDMigration = (data: CephClusterKind) => {
```

Comment:
```suggestion
const calculateOSDMigration = (: CephClusterKind) => {
```
dont use name 'data' everywhere, use the proper name

Reply:
done